sessionctx/variable/tidb_vars.go

```diff
@@ -892,6 +892,7 @@ const (
 	DefTiDBEnablePrepPlanCache                   = true
 	DefTiDBPrepPlanCacheSize                     = 100
 	DefTiDBPrepPlanCacheMemoryGuardRatio         = 0.1
+	DefTiDBEnableConcurrentDDL                   = true
```

Comment:
Maybe default enable it after all test is better.

Reply:
enable it by default is helpful for running unit-test, and I think it does not matter since there is no new tidb release before delivery of the feature.

Reply:
Yes, We can create a new pipeline to enable this switch to test.

Reply:
If you plan to just change the default for tests, see https://github.com/pingcap/tidb/pull/34755 (it gets a bit cleaner with this PR).